### PR TITLE
Adds a script to install service binaries from Cattle jar.

### PR DESCRIPTION
**Pull request by cloudnautique**
This change unpacks the jar file in a temporary location, and reads
from the cattle-global.properties file for service.package.*.url
entries. It will then curl those.

### CODE REVIEW

**Comment by cjellick** [comment]
LGTM!

**Comment by cloudnautique** [comment]
ping @ibuildthecloud

File: server/artifacts/install_cattle_binaries [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    if [ "$xz" = "true" ]; then
+        tar -xpJf ${file} -C ${dest}
+    fi
+    chmod +x ${dest}/${file}
+}
+
+if [ -f "/usr/share/cattle/cattle.jar" ]; then
+    unzip /usr/share/cattle/cattle.jar
+    mkdir -p res && cd res
+    unpack200 ../WEB-INF/lib/cattle-resources-0.5.0-SNAPSHOT.jar.pack resources.jar
```
Comment (ibuildthecloud) [inline server/artifacts/install_cattle_binaries]: Probably should change to `./WEB-INF/lib/cattle-resources-*.jar.pack`

File: server/Dockerfile [hunk 2]
```diff
@@ -5,7 +5,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends openjdk-7-jre-h
                     curl \
                     mysql-server \
                     xz-utils \
-                    keychain
+                    keychain \
+                    unzip
```
Comment (ibuildthecloud) [inline server/Dockerfile]: Why add unzip?  I'm guessing `jar` exists already in the image.

Comment (cloudnautique) [inline server/Dockerfile]: It did not. I checked.

**Pull request merged**